extensions/adobe/experience/decisioning/decisionevent.example.1.json

```diff
@@ -2,62 +2,65 @@
   "xdm:timestamp": "2019-01-26T15:52:25+00:00",
   "xdm:identityMap": {
-    "ECID": [
-      { "xdm:id": "https://data.adobe.io/entities/identity/49312748749132" }
-    ]
-  },
-  "xdm:device": {
-    "xdm:typeID": "IMac",
-    "xdm:typeIDService": "https://ns.adobe.com/xdm/external/adobecampaign",
-    "xdm:type": "desktop",
-    "xdm:manufacturer": "Apple"
-  },
-  "xdm:placeContext": {
-    "xdm:localTime": "2017-09-26T15:52:25+13:00",
-    "xdm:geo": {
-      "@id": "https://data.adobe.io/entities/geo/tokyo",
-      "xdm:countryCode": "JP",
-      "xdm:stateProvince": "JP-13",
-      "xdm:city": "Tōkyō",
-      "xdm:postalCode": "141-0032",
-      "schema:latitude": 35.6185,
-      "schema:longitude": 139.73237
-    }
-  },
-  "xdm:web": {
-    "xdm:webPageView": {
-      "xdm:URL": "https://www.example.com"
-    },
-    "xdm:webReferrer": {
-      "xdm:URL": "https://www.examplereferrer.com/",
-      "xdm:domain": "examplereferrer.com"
-    }
+    "ECID": [{ "xdm:id": "33441528584054496761339722935948080609" }]
   },
   "https://ns.adobe.com/experience/decisioning/propositionID": "3cc33a7e-13ca-4b19-b25d-c816eff9a70a",
-  "https://ns.adobe.com/experience/decisioning/propositionDetails": [
+  "https://ns.adobe.com/experience/decisioning/propositions": [
     {
-      "xdm:activity": {
-        "xdm:id": "xcore:offer-activity:ebc48132c26ccfc"
+      "xdm:scopeDetails": {
+        "xdm:placement": {
+          "xdm:id": "xcore:offer-placement:c652463157c2aa1",
+          "repo:etag": "4"
+        },
+        "xdm:activity": {
+          "xdm:id": "xcore:offer-activity:ebc48132c26ccfc",
+          "repo:etag": "8"
+        },
+        "xdm:strategy": [
+          {
+            "xdm:step": "ranking",
+            "xdm:strategyID": "ranking-strategy:7251468357c2236",
+            "xdm:algorithmID": "ts",
+            "xdm:trafficType": "noncontextual"
+          }
+        ],
+        "xdm:experience": {
+          "xdm:id": "ExperienceA"
+        }
       },
-      "xdm:selections": [
+      "xdm:items": [
         {
-          "xdm:id": "xcore:personalized-offer:e526a1503bf29e9",
-
-          "https://ns.adobe.com/experience/decisioning/propositionsTotal": {
+          "xdm:id": "xcore:personalized-offer:f67bab756ed6ee4",
+          "repo:etag": "5",
+          "xdm:score": 83,
+          "xdm:propositionsTotal": {
             "xdm:value": 948
           },
-          "https://ns.adobe.com/experience/decisioning/propositionsProfile": {
+          "xdm:propositionsProfile": {
             "xdm:value": 2
           }
+        },
+        {
+          "xdm:id": "xcore:personalized-offer:e91ee850a0bb7d9",
+          "repo:etag": "3",
+          "xdm:score": 12
         }
       ]
     },
     {
-      "xdm:activity": {
-        "xdm:id": "xcore:offer-activity:f203512e02542b5"
-      },
-      "xdm:fallback": {
-        "xdm:id": "xcore:fallback-offer:e91ce7243fd8c2a"
+      "xdm:scopeDetails": {
```

Comment:
This second example only has the `xdm:scopeDetails` node, and no `xdm:items` node. Is that a valid proposition?

Reply:
Ty, I missed that, yes we want at least one item in the array (the schema says so :)
It is corrected now.